test/js/components/tasks/detail.test.jsx

```diff
@@ -380,7 +380,9 @@ describe('<TaskDetail/>', () => {
       });
       fireEvent.click(getByText('Submit Task for Review'));
 
-      expect(getByText('Submit this task for review')).toBeVisible();
+      getAllByText('Submit Task for Review').forEach((element) => {
+        expect(element).toBeVisible();
+      });
```

Comment:
I'm not clear why this test changed.

Reply:
The text was visible multiple times and so `getByText` returned an error.

Reply:
Again, I don't think we're _intentionally_ changing user-facing text for the task-detail page in this PR, so if the test is failing I think that represents a bug we want to surface.